### PR TITLE
6480 webjar dependencies

### LINKED ISSUE #6480: update webjars dependency
Techdebt:

Update webjars dependency to latest

### PR DESCRIPTION
**Pull request by TipzCM**
closes #6480

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2333,11 +2333,6 @@
 				<artifactId>assertj-core</artifactId>
 				<version>3.22.0</version>
 			</dependency>
-			<dependency>
-				<groupId>org.assertj</groupId>
-				<artifactId>assertj-json</artifactId>
-				<version>3.22.0</version>
-			</dependency>
```
Comment (tadgh) [inline pom.xml:2340]: question: why the removal?

File: hapi-fhir-jpaserver-test-r5/src/test/java/ca/uhn/fhir/jpa/dao/r5/UpliftedRefchainsAndChainedSortingR5Test.java [hunk 2]
```diff
@@ -296,10 +296,9 @@ public void testCreate_InTransaction_TargetCreated_RefsUsePlaceholderIds() {
 		assertEquals(1, myCaptureQueriesListener.countSelectQueriesForCurrentThread());
 
 		// Verify correct indexes are written
+		logAllStringIndexes();
 
 		runInTransaction(() -> {
-			logAllStringIndexes();
```
Comment (tadgh) [inline hapi-fhir-jpaserver-test-r5/src/test/java/ca/uhn/fhir/jpa/dao/r5/UpliftedRefchainsAndChainedSortingR5Test.java:301]: question: why this change?

**Review by tadgh — APPROVED** [review]
Approved pending questions

File: hapi-fhir-jpaserver-test-r5/src/test/java/ca/uhn/fhir/jpa/dao/r5/UpliftedRefchainsAndChainedSortingR5Test.java (hunk 2 above)
Comment (TipzCM) [reply hapi-fhir-jpaserver-test-r5/src/test/java/ca/uhn/fhir/jpa/dao/r5/UpliftedRefchainsAndChainedSortingR5Test.java:301]: fix intermittents.

"logAllStringIndexes" is. itself, already wrapped in a transaction. and the nested transaction was causing a bunch of failures i brought up on the intermittent channel
